examples/jms-durable-topic-message-subscriber/jms-durable-topic-message-subscriber.bal

```diff
@@ -23,9 +23,10 @@ endpoint jms:DurableTopicSubscriber subscriber {
 // Bind the created subscriber to the listener service.
 service<jms:Consumer> jmsListener bind subscriber {
 
-    // OnMessage resource get invoked when a message is received.
+    // The `OnMessage` resource gets invoked when a message is received.
     onMessage(endpoint consumer, jms:Message message) {
-        // Retrieve the text message content with error lifting by using the check keyword.
+        // Retrieve the text message content using the `check` keyword. `Check` terminates the function if an error
```

Comment:
"Retrieve the text message content using the `check` keyword" is not correct. Check is just error lifting. Above statement means the text message is retrieved using the check keyword.